q2_assembly/quast/quast.py

```diff
@@ -159,13 +161,20 @@ def _evaluate_contigs(
     if references:
         all_refs_dir = os.path.join(results_dir, "references")
         os.makedirs(all_refs_dir, exist_ok=True)
+        copy_tree(str(references), all_refs_dir)
```

Comment:
Why do you need to copy this? This was required when we had files containing multiple genomes per fasta file which had to be split into individual files. Your genomes already are separated so you shouldn't need to copy anything - you just need to gather paths to those and pass them to quast below.